Source/WebCore/page/FocusController.cpp

```diff
@@ -965,8 +965,6 @@ void FocusController::setIsVisibleAndActiveInternal(bool contentIsVisible)
         for (auto& scrollableArea : *scrollableAreas) {
             if (!scrollableArea->scrollbarsCanBeActive())
                 continue;
-            ASSERT(m_page.shouldSuppressScrollbarAnimations());
```

Comment:
The assertion was added in https://bugs.webkit.org/show_bug.cgi?id=96158, so I think we need to understand why it's OK to remove it.

Reply:
I believe the logic of this assert was that either the scrollbars of each scrollable area should be active or the api was used to suppress the scrollbar animations. Now these two are not correlated as the scrollbars could be not active for reasons other than the api. Since this logic no longer holds true, I think it's ok to delete the assert.

Reply:
Reverting to original assert after some investigation.